LEGO1/lego/sources/misc/legoweedge.h

```diff
@@ -0,0 +1,45 @@
+#ifndef __LEGOWEEDGE_H
+#define __LEGOWEEDGE_H
+
+#include "decomp.h"
+#include "legotypes.h"
+
+class LegoWEEdge;
+
+// SIZE 0x24
+class Edge {
```

Comment:
Lacks `VTABLE` annotation (and possibly stubs for virtual functions if there are any)